Current/Max degree of parallelism.sql

```diff
@@ -10,13 +10,24 @@ BEGIN
 
     SET NOCOUNT ON
 
-    Declare @Output VarChar(max)
+    Declare @Output VarChar(max), @Warning VarChar(max)
     Set @Output = ''
+    Set @Warning = 'Warning: Max degree of parallelism is setup to use all cores'
 
-    select @Output = 'Warning: Max degree of parallelism is setup to use all cores'
-    from   sys.configurations
-    where  name = 'max degree of parallelism'
-           and value_in_use = 0
+    IF (SERVERPROPERTY('EngineEdition') = 5)
```

Comment:
Have you checked whether this is also the case on Managed Instances (which have a different EngineEdition)?

Reply:
Not yet, this was just to support Azure SQL Databases.